src/adapter/src/rbac.rs

```diff
@@ -145,8 +145,9 @@ pub fn check_plan(
     }
 
     // Validate that the current session has the required attributes to execute the provided plan.
+    // Note: role attributes are not inherited by role membership.
```

Comment:
Wait what? Is this because of NOINHERIT or something? I'm reading over https://www.postgresql.org/docs/current/role-membership.html but don't see what's going on here.